Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py

```diff
@@ -1414,7 +1414,7 @@ def send_usage_data(auth_token, account_id, client_id, msg_type, USE_SSL):
         "integration_code": "TA-CX",
         "integration_name": "Cortex XSOAR",
         "integration_app": "Malwarebytes",
```

Comment:
`integration_app` should still remain `Malwarebytes`?

Reply:
yeah, that should be threatdown. I have made the changes, thanks for pointing out.